mathics/builtin/attributes.py

```diff
@@ -173,6 +191,19 @@ class Unprotect(Builtin):
         'Unprotect[symbols__]': 'ClearAttributes[{symbols}, Protected]',
     }
 
+    def apply(self, pattern, evaluation):
+        "Unprotect[pattern_]"
```

Comment:
The defined behavior is different from the established in the docstring: this does not accept a list of patterns, but just one.  
,

Reply:
Thanks for pointing out. 

Yes, that should probably be `pattern__` and the code will have to get more complicated too. 

I got the docstring from Mathematica. And I doubt the code handles _form_ like it should as well.

However, the particular problem I wanted to fix was `Unprotect["\[GraveAccent]*]` though which explains the discrepancy. 

I'll address this when I have a chance to come back to it.